plugins/SegmentEditor/SegmentEditor.php

```diff
@@ -80,7 +80,19 @@ public function addSystemSummaryItems(&$systemSummary)
         $storedSegments = StaticContainer::get('Piwik\Plugins\SegmentEditor\Services\StoredSegmentService');
         $segments = $storedSegments->getAllSegmentsAndIgnoreVisibility();
         $numSegments = count($segments);
-        $systemSummary[] = new SystemSummary\Item($key = 'segments', Piwik::translate('CoreHome_SystemSummaryNSegments', $numSegments), $value = null, $url = null, $icon = 'icon-segment', $order = 6);
+
+        $numAutoArchiveSegments = 0;
+        $numRealTimeSegments = 0;
+        foreach ($segments as $segment) {
+            $autoArchive = (int)$segment['auto_archive'];
+            if (!empty($autoArchive)) {
+                ++$numAutoArchiveSegments;
+            } else {
+                ++$numRealTimeSegments;
+            }
+        }
+
+        $systemSummary[] = new SystemSummary\Item($key = 'segments', Piwik::translate('CoreHome_SystemSummaryNSegments', [$numSegments, $numAutoArchiveSegments, $numRealTimeSegments]), $value = null, $url = null, $icon = 'icon-segment', $order = 6);
```

Comment:
if `browser_archiving_disabled_enforce=1 (Rules::isBrowserArchivingAvailableForSegments()=true)` then we wouldn't show the message of how many are pre-processed/ real time and only show overall number of segments. As most users wouldn't understand what pre-processed etc means (eg on cloud). In that case all segments would be pre-processed

We would then only like previously show the total number of segments.

Reply:
assuming you mean `Rules::isBrowserArchivingAvailableForSegments()==false` since isBrowserArchivingAvailableForSegments has `!$generalConfig['browser_archiving_disabled_enforce']`

Reply:
👍